### DISCUSSION TITLE
Wayland support

### DISCUSSION THREAD

**Discussion by notwidow** [qtile/qtile#4029] [Q&A] [answered]
Hi, Is qtile working well on wayland without issues? If it has all features . i would love to use it.

**Comment by elParaguayo** [comment]
Not sure what "all features" means but your best bet is just to give it a try and see if you like it!

**Reply by notwidow** [reply]
> Not sure what "all features" means but your best bet is just to give it a try and see if you like it!

i once checked the qtile some time ago and there were todos for wayland to be completed so asking about those... todo list for wayland..

**Comment by m-col** [comment, marked as answer]
Hey! The to do list can be found here: https://github.com/qtile/qtile/discussions/2409

There are plenty of us who use Wayland as our daily driver, so it can be done! :)

Now, that isn't to say there aren't bugs, there are, but it is usable. However if you have a Nvidia GPU you will likely have problems. You can see reported bugs using the 'wayland' issues tag: https://github.com/qtile/qtile/issues?q=is%3Aissue+is%3Aopen+label%3AWayland

Also the IRC/discord/matrix room is quite active so feel free to ask there for any help if you'd prefer.

**Reply by notwidow** [reply]
> Hey! The to do list can be found here: #2409
> 
> There are plenty of us who use Wayland as our daily driver, so it can be done! :)
> 
> Now, that isn't to say there aren't bugs, there are, but it is usable. However if you have a Nvidia GPU you will likely have problems. You can see reported bugs using the 'wayland' issues tag: https://github.com/qtile/qtile/issues?q=is%3Aissue+is%3Aopen+label%3AWayland
> 
> Also the IRC/discord/matrix room is quite active so feel free to ask there for any help if you'd prefer.

i do have nvidia but also intel... and i like the nvidia but i use intel for its open source drivers. and thanks . i will try nvidia qtile tomorrow.. btw coming from sway...  been using sway for about a month and getting disappointed everyday from sway due to bugs and bugs and nothing else. my arch got totally broken... and now i hate sway

**Reply by m-col** [reply]
Hmm well Qtile almost certainly has more bugs than sway. Qtile has a bit of a learning curve to get things set up, because it gives the user a lot of power via the Python config file, to do both great things and bad things! Hopefully it works smoothly for you.

**Reply by notwidow** [reply]
> Hmm well Qtile almost certainly has more bugs than sway. Qtile has a bit of a learning curve to get things set up, because it gives the user a lot of power via the Python config file, to do both great things and bad things! Hopefully it works smoothly for you.

by the way what is the key for getting all other key bindings? i once saw a video in which he said a keybinding and that displayed all other keybindings as well....
